daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -812,6 +812,24 @@ class RazerBlackWidow2019(_RippleKeyboard):
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1501/1501-blackwidow2019.png"
 
 
+class RazerBlackWidowEssential(_RippleKeyboard):
+    """
+    Class for the Razer BlackWidow Essential
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_BlackWidow_Essential(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0237
+    HAS_MATRIX = True
+    MATRIX_DIMS = [6, 22]
+    METHODS = ['get_device_type_keyboard', 'set_static_effect', 'set_none_effect',
+               'set_breath_single_effect', 'set_key_row', 'get_game_mode',
+               'set_game_mode', 'get_macro_mode', 'set_macro_mode', 'get_macro_effect',
+               'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro']
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1501/1501-blackwidow2019.png"
```

Comment:
Please find a picture for your device on https://support.razer.com (or if not possible, somewhere else but hosted by Razer)

Reply:
I can't find the keyboard on razer's site.. neither on any razer's domains, I just keep getting images from others stores, any other suggestions?

Reply:
Hmm... Maybe find a keyboard which looks more or less like this one and use the picture of that? And add a comment, that there aren't any pictures of this device available or sth

Reply:
Ok, after digging a little bit, the blackwidow-2019 is the most similar keyboard, the main differences are the RGB lights, in the Essential one, it's only green also it's has less lights effects. I'll add a comment then. Thanks!

Reply:
this is still missing

Reply:
I found the device on the Chinese Razer store :thinking: http://cn.razerzone.com/gaming-keyboards/razer-blackwidow-essential Doesn't have the pictures I'm looking for though. Let's keep it with that for now then.